### PR TITLE
Add release notes for `edge-21.9.1`

### PR DESCRIPTION
**Pull request by Pothulapati**
This release includes various improvements and feature additions across the policy
feature i.e, New validating webhook for policy resources. This also includes changes
in the proxy i.e, terminating TCP connections when a authorization is revoked, improvements
in the proxy authorization metrics. In addition, proxy injector has also been updated
to set the right `opaque-ports` annotation on services with default opaque ports.

* Added a new validating admission controller to validate the policy resources
* Updated the proxy-init to remove a rule which caused the packets from the proxy
  with destination != 127.0.0.1 on localhost to be sent to the inbound proxy
* Updated inbound policy enforcement to interrupt TCP forwarding if a previously
  established authorization is revoked
* Added new proxy metrics to expose authorization decisions
* Updated inbound TCP metrics to only include a `srv_name` label
* Updated the proxy to export route-oriented metrics only when a ServiceProfile
  is enabled
* Updated the proxy's release build configuration to improve CPU and memory utilization 
* Added DNS name validation to the `proxy-identity` binary which creates the
  read-only private key required by the proxy (thanks @yorkijr!)
* Updated the identity controller's default policy to be `cluster-unauthenticated`
* Updated the proxy injector to include the correct default ports as opaque with
  services
* Deprecated the usage of `vis stat ts` and print a warning about the SMI extension
* Updated various dependencies across the dashboard, policy-controller
  (thanks @dependabot!)

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,5 +1,35 @@
...
 
+## edge-21.9.1
+
+This release includes various improvements and feature additions across the policy
+feature i.e, New validating webhook for policy resources. This also includes changes
+in the proxy i.e, terminating TCP connections when a authorization is revoked, improvements
+in the proxy authorization metrics. In addition, proxy injector has also been updated
+to set the right `opaque-ports` annotation on services with default opaque ports.
+
+* Added a new validating admission controller to validate the proxy resources
```
Comment (mateiidavid) [inline CHANGES.md:11]: would it be better to say we validate policy resources here?

File: CHANGES.md [hunk 2]
```diff
@@ -1,5 +1,35 @@
...
+  with destination != 127.0.0.1 on localhost to be sent to the inbound proxy
+* Added new `LINKERD_DOCKER_REGISTRY` env variable to configure the docker
+  registry in the CLI
+* Updated inbound policy enforcement to interrupt TCP forwarding if a previously
+  established authorization is revoked
+* Added new proxy metrics w.r.t Authorization along with traffic target labels
+* Updated inbound TCP metrics to only include a `srv_name` label
+* Updated the proxy to export route-oriented metrics only when a ServiceProfile
+  is enabled.
+* Updated tokio to include performance improvements and to enable link-time
```
Comment (mateiidavid) [inline CHANGES.md:22]: nit: do you think we should exclude this? doesn't seem very user facing.

File: CHANGES.md [hunk 3]
```diff
@@ -1,5 +1,35 @@
...
+  optimizations in the release builds
+* Added DNS name validation to the `proxy-identity` binary which creates the
+  read-only private key required the proxy (thanks @yorkijr!)
```
Comment (mateiidavid) [inline CHANGES.md:25]:
```suggestion
  read-only private key required by the proxy (thanks @yorkijr!)
```

just a little typo 😳

**Review by mateiidavid — APPROVED** [review]
Left a few comments but nothing major, the changelog looks good to me! 🚢 📦

File: CHANGES.md (hunk 1 above)
Comment (Pothulapati) [reply CHANGES.md:11]: yeah, I wanted to write policy

File: CHANGES.md (hunk 2 above)
Comment (Pothulapati) [reply CHANGES.md:22]: Not very user facing but perf improvements felt important to be mentioned! 🤔

Comment (Pothulapati) [reply CHANGES.md:22]: Hmm, We can remove the link-time part I guess. Updating

File: CHANGES.md [hunk 4]
```diff
@@ -1,5 +1,34 @@
...
+feature i.e, New validating webhook for policy resources. This also includes changes
+in the proxy i.e, terminating TCP connections when a authorization is revoked, improvements
+in the proxy authorization metrics. In addition, proxy injector has also been updated
+to set the right `opaque-ports` annotation on services with default opaque ports.
+
+* Added a new validating admission controller to validate the policy resources
+* Updated the proxy-init to remove a rule which caused the packets from the proxy
+  with destination != 127.0.0.1 on localhost to be sent to the inbound proxy
+* Added new `LINKERD_DOCKER_REGISTRY` env variable to configure the docker
+  registry in the CLI
```
Comment (olix0r) [inline CHANGES.md:15]:
```suggestion
```

I don't think this is something we really want to advertise as a feature. It's more of a hack for our CI.

File: CHANGES.md [hunk 5]
```diff
@@ -1,5 +1,34 @@
...
+* Updated inbound policy enforcement to interrupt TCP forwarding if a previously
+  established authorization is revoked
+* Added new proxy metrics w.r.t Authorization along with traffic target labels
```
Comment (olix0r) [inline CHANGES.md:18]:
```suggestion
* Added new proxy metrics to expose authorization decisions
```

File: CHANGES.md (hunk 2 above)
Comment (mateiidavid) [reply CHANGES.md]: I'm good either way, I see your point too. Just asked to see what you think :)

File: CHANGES.md [hunk 6]
```diff
@@ -1,5 +1,34 @@
...
+* Updated inbound TCP metrics to only include a `srv_name` label
+* Updated the proxy to export route-oriented metrics only when a ServiceProfile
+  is enabled.
+* Updated the tokio dependency in the proxy to include performance improvements
```
Comment (olix0r) [inline CHANGES.md:22]:
```suggestion
* Updated the proxy's release build configuration to improve CPU and memory utilization 
```

File: CHANGES.md [hunk 7]
```diff
@@ -1,5 +1,34 @@
...
+* Updated the proxy-init to remove a rule which caused the packets from the proxy
+  with destination != 127.0.0.1 on localhost to be sent to the inbound proxy
+* Added new `LINKERD_DOCKER_REGISTRY` env variable to configure the docker
+  registry in the CLI
+* Updated inbound policy enforcement to interrupt TCP forwarding if a previously
+  established authorization is revoked
+* Added new proxy metrics w.r.t Authorization along with traffic target labels
+* Updated inbound TCP metrics to only include a `srv_name` label
+* Updated the proxy to export route-oriented metrics only when a ServiceProfile
+  is enabled.
```
Comment (olix0r) [inline CHANGES.md:21]:
```suggestion
  is enabled
```

**Review by olix0r — APPROVED** [review]

File: CHANGES.md [hunk 8]
```diff
@@ -1,5 +1,34 @@
 # Changes
 
+## edge-21.9.1
+
+This release includes various improvements and feature additions across the policy
+feature i.e, New validating webhook for policy resources. This also includes changes
+in the proxy i.e, terminating TCP connections when a authorization is revoked, improvements
+in the proxy authorization metrics. In addition, proxy injector has also been updated
```
Comment (kleimkuhler) [inline CHANGES.md:8]:
```suggestion
in the proxy authorization metrics. In addition, the proxy injector has also been updated
```

File: CHANGES.md [hunk 9]
```diff
@@ -1,5 +1,34 @@
...
+* Added new proxy metrics w.r.t Authorization along with traffic target labels
+* Updated inbound TCP metrics to only include a `srv_name` label
+* Updated the proxy to export route-oriented metrics only when a ServiceProfile
+  is enabled.
+* Updated the tokio dependency in the proxy to include performance improvements
+* Added DNS name validation to the `proxy-identity` binary which creates the
+  read-only private key required by the proxy (thanks @yorkijr!)
+* Updated the identity controller's default policy to be `cluster-unauthenticated`
+* Updated the proxy injector to include the correct default ports as opaque with
+  services
```
Comment (kleimkuhler) [inline CHANGES.md:25]: wording:
```suggestion
* Updated the proxy injector to add only the default opaque ports that a pod or service exposes
```

**Review by kleimkuhler — APPROVED** [review]

**Pull request merged**